src/TextField/TextField.js

```diff
@@ -59,6 +62,10 @@ function TextField(props) {
         inputRef={inputRef}
         {...InputProps}
       />
+      {helperText &&
+        <FormHelperText className={helperTextClassName}>
```

Comment:
Note for self, in the future, we might expose a `FormHelperTextProps` as well as `InputLabelProps` so people can have access to it without needed to use the lower level components (we already do it with `InputProps`). If you want to do it feel free 😄 .